Generic Types, Traits and Lifetime/Generic Types, Traits and Lifetime/Trait Bound Syntax/task-info.yaml

```diff
@@ -0,0 +1,26 @@
+type: edu
+files:
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 45
+    length: 12
+    placeholder_text: // Something goes here
+  - offset: 75
+    length: 1
+    placeholder_text: f32
+  - offset: 140
+    length: 12
+    placeholder_text: // Something goes here
```

Comment:
An open placeholder with "//" affects all of the remaining line, leaving following stuff commented. I suggest using a "/**/" inline placeholder to avoid this